Add warning section in database migration (#545)
CleverCloud/doc.clever-cloud.com: administrate/database-migration.md

## Migrating any database

{{< alert "warning" "Read only users" >}}

>     As we do not support the migration of read only users, if you have created read only users you will have to create new ones.

Reviewer: ```suggestion
    We do not support the migration of read only users. If you have any, you will have to create new ones at the end of the migration.
```
